job/job.go

```diff
@@ -34,6 +37,20 @@ type Job struct {
 	instance *packet.Instance
 }
 
+// isHardwareHavingActiveWorkflow fetches workflows for a given hwID and returns
+// the status true if there is a pending (active) workflow for hwID
+// hwID is the hardware/worker ID corresponding to the MAC
+func isHardwareHavingActiveWorkflow(wcl *tw.WorkflowContextList) (bool, error) {
+
+	for _, wf := range (*wcl).WorkflowContexts {
+		if wf.CurrentActionState == 0 {
```

Comment:
Please don't use a magic number here, use the ActionState enum

Reply:
0 is harcoded for pending on workflow.pb.go, do you want to replace the '0' with constant from workflow.pb.go , `ActionState_ACTION_PENDING`  or use map for `ACTION_PENDING` `ActionState_value` ?

Reply:
done